src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -15,6 +15,7 @@ Microsoft.CodeAnalysis.Workspace.RaiseTextDocumentClosedEventAsync(Microsoft.Cod
 Microsoft.CodeAnalysis.Workspace.RaiseTextDocumentOpenedEventAsync(Microsoft.CodeAnalysis.TextDocument document) -> System.Threading.Tasks.Task
 Microsoft.CodeAnalysis.Workspace.TextDocumentClosed -> System.EventHandler<Microsoft.CodeAnalysis.TextDocumentEventArgs>
 Microsoft.CodeAnalysis.Workspace.TextDocumentOpened -> System.EventHandler<Microsoft.CodeAnalysis.TextDocumentEventArgs>
+Microsoft.CodeAnalysis.Shared.TestHooks.IAsyncToken
```

Comment:
>IAsyncToken [](http://example.com/codeflow?start=40&length=11)

Does this need to be public?

Reply:
I assume that nothing with "TestHooks" in the name does. The PublicAPI files have been left in various states in the interest or expediency while I'm making larger changes to the API surface, I'll be resolving those one way or another before this leaves PR.